### DISCUSSION TITLE
Problems with chrome user data logins

### DISCUSSION THREAD

**Discussion by ga-it** [ArchiveBox/ArchiveBox#949] [General]
FILED BUG UNDER ISSUES

Hi

I wish to use ArchiveBox to save content from subscriptions behind paywalls.

I have exported cookies in Netscape format (for WGET I understand - COOKIES_FILE) and shared Chromium paths with Archivebox (CHROME_USER_DATA_DIR).

Neither have resulted in logins (evident in screenshots and denied access).

The latter Chromium user data folder process has been particularly problematic - rejecting the path provided (no "Default" profile found where it was there) before suddenly accepting it.

To ensure a usable cookie, I created a VNC session on the server, browsed via Chromium to the site, logged in and then tried using archivebox and providing the path to the User Data folder.

Great project - but the login feature is critical for me to archive content behind paywalls.

Regards

Marc

ArchiveBox Dev docker image running on Debian Testing

**Comment by pirate** [comment]
Closing this as a duplicate of #952, no need to open two separate tickets/discussions.
